src/PostgREST/Main.hs

```diff
@@ -105,7 +99,8 @@ main = do
   -- print $ findRelation (fakeRels ++ allRels) "test" "pg_source" "clients"
 
   runSettings appSettings $ middle $ \ req respond -> do
+    time <- getPOSIXTime
     body <- strictRequestBody req
     resOrError <- liftIO $ H.session pool $ H.tx txSettings $
-      runWithClaims conf (app dbstructure conf body) req
+      runWithClaims conf time (app dbstructure conf body) req
```

Comment:
I don't think this `time` will advance as the server runs. So it will always check for expiration based on server start time.